gen3utils/etl/etl_validator.py

```diff
@@ -364,6 +366,13 @@ def check_mapping_constraints(mappings, model, recorded_errors, underscore):
                     index,
                     nodes_for_category,
                 )
+        if "project_id" not in index.props and m.get("type") == "aggregator":
```

Comment:
is `project_id` not required for "collector" mappings?

Reply:
I think so, because we can collect some file  which not belonging any project @paulineribeyre

Reply:
oh ok!